_topic_maps/_topic_map.yml

```diff
@@ -2056,6 +2056,9 @@ Topics:
 - Name: Provisioning and deploying a distributed unit (DU)
   File: cnf-provisioning-and-deploying-a-distributed-unit
   Distros: openshift-webscale
+- Name: Deploying clusters manually on a small number of single nodes as a distributed unit
```

Comment:
Suggest changing title to "Deploying Distributed Manually on Single Node Openshift"

Reply:
How about "Deploying distributed units (DUs) manually in single nodes" (We want to avoid "Single Node Openshift")

Reply:
Never mind my comment - will use yours.